style.css

```
@@ -65,6 +65,12 @@ h6 a:hover {
   line-height: var(--line-height-s--font-family-tertiary);
 }
 
+/* GPAP typography style overides */
+
+h1 {
+  text-transform:uppercase;
+}
```

```suggestion
h1 {
  text-transform: uppercase;
}
```

This is great, we should just be adding spaces here to keep it consistent with the rest of the code in this file.